### PR TITLE
BaSP-TG-142: Super Admin Home

### PR DESCRIPTION
**Pull request by Ansitox**
- In Home add admins Table
- Create Admins only button
- Edit Admins only button
- Add search shared component
- Delete Admin button and functionality (soft delete)
- When click create admin button in the table, redirect to create Admin form
- When click edit admin button in the table, redirect to edit Admin form

QA:
How to test?:
Log in as a Super Admin (email: oclacep@wikimedia.org, password: prueba1234)
When logged you should se a nav bar (temporary)
In the Home page you should see a list of Admins
The "+" button redirects you to a form page (empty)
The "edit" buttons should redirect you to the same form, but the path should have the id of the admin to edit.
The "delete" button should open a modal, on cancel modal will close, on confirm a message modal will appear with the error/success message, then will close.
Note: The nav bar is temporaryand just to use on the test of the PRs, any bad function on navigation should be ignored.

### CODE REVIEW

File: src/Components/SuperAdmins/Home/index.js [hunk 1]
```diff
@@ -1,7 +1,168 @@
...
+          value.startDate?.toLowerCase().includes(search.toLocaleLowerCase()) ||
+          value.endDate?.toLowerCase().includes(search.toLocaleLowerCase()) ||
+          value.employees?.toLowerCase().includes(search.toLocaleLowerCase())
+      );
+
+  const onConfirm = () => {
+    if (modalContent.content.includes('logout')) {
+      dispatch(logout());
+    } else {
+      console.log(itemId);
```
Comment (matescarabino) [inline src/Components/SuperAdmins/Home/index.js:53]: Is this necessary?

**Review by matescarabino — CHANGES_REQUESTED** [review]

**Review by CristianBucci — APPROVED** [review]
Everything looks right!

**Review by matescarabino — APPROVED** [review]
LGTM

**Review by julianpiaggio — APPROVED** [review]
🕵️‍♂️ QA FEEDBACK

We wanted to tell you that we consider that your pr complies
the requirements to be approved, and that it can be merged into the master branch 👍

**Pull request merged**